scripts/docs/build-docs.js

```diff
@@ -27,7 +27,10 @@ function buildDocs() {
 	if ( skipApi ) {
 		promise = Promise.resolve();
 	} else {
-		promise = buildApiDocs();
+		promise = buildApiDocs()
+			.catch( () => {
```

Comment:
We can miss a random error easily with such catching.

Reply:
I didn't find a better way.

Reply:
even the `catch( err => { console.error( err ); process.exitCode = 1; } )` will be better.